plugins/woocommerce-admin/client/customer-effort-score-tracks/constants.ts

```diff
@@ -0,0 +1,5 @@
+export const SHOWN_FOR_ACTIONS_OPTION_NAME =
+	'woocommerce_ces_shown_for_actions';
+export const ADMIN_INSTALL_TIMESTAMP_OPTION_NAME =
+	'woocommerce_admin_install_timestamp';
```

Comment:
I couldn't find the place where these two `const` are being used.

Reply:
Aaah yeah, I kinda copy pasted them from another PR, that one is merged now, so all of them should be used in a couple other files now.